### DISCUSSION TITLE
How Do I Fix Camera

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#716] [Q&A]
Ok So I Wanted To Fix Them Camera But When IT Heads To Link His Camera Is Messed Up 
![image](https://user-images.githubusercontent.com/82290120/120563710-822f2200-c3c6-11eb-86fb-042d0a83a182.png)
How Do I Fix This HELP!

**Comment by FlashlightET** [comment]
How
